app/views/taggings/_related_item_tagging_entry.html.erb

```diff
@@ -0,0 +1,23 @@
+<li class="related-content-item-entry">
+  <% unless base_path.blank? || @tagging_update.related_item_errors[base_path] %>
+    <div class="title select2-search-choice ui-sortable-handle ">
+      <a href="https://www.gov.uk<%= base_path %>" class="js-artefact-name">
```

Comment:
Should be, but haven't looked into this yet. Could always add a helper if one doesn't exist.